azure-pipelines.yml

```diff
@@ -112,5 +111,77 @@ jobs:
         platform: '$(buildPlatform)'
         configuration: '$(buildConfiguration)'
 
-    - publish: 'src/WebJobs.Extensions.DurableTask/bin'
-      artifact: bin
\ No newline at end of file
+    - task: EsrpCodeSigning@1
```

Comment:
Now that we have an official release pipeline, let's remove the signing from this one and do signing only for official releases. That way we're not spending time and resources doing signing for a package that doesn't really require it.